### ISSUE TITLE
[A11y] Address A11y issues in Header

### ISSUE THREAD

**Issue by jenny-s51** [opendatahub-io/odh-dashboard#1101]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Certain Aria roles must contain particular children:
![image](https://user-images.githubusercontent.com/32821331/231211186-3d471856-6c9f-47fa-adb3-4cd3ae27e81f.png)

### Expected Behavior

Resolve critical a11y header violation.

**Comment by jenny-s51** [comment]
Closing for now as this is raising the same axe error in PF. Related issue: https://github.com/patternfly/patternfly-react/issues/8939.
